presto-hive-hadoop2/src/test/java/io/prestosql/plugin/hive/TestHiveAlluxioMetastore.java

```diff
@@ -0,0 +1,233 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.hive;
+
+import alluxio.client.table.TableMasterClient;
+import alluxio.conf.PropertyKey;
+import io.prestosql.plugin.hive.metastore.HiveMetastore;
+import io.prestosql.plugin.hive.metastore.alluxio.AlluxioHiveMetastore;
+import io.prestosql.plugin.hive.metastore.alluxio.AlluxioHiveMetastoreConfig;
+import io.prestosql.plugin.hive.metastore.alluxio.AlluxioMetastoreModule;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Parameters;
+
+public class TestHiveAlluxioMetastore
+        extends AbstractTestHive
+{
+    private static final String SCHEMA = "default";
+
+    private String alluxioAddress;
+
+    @Parameters({
+            "hive.hadoop2.alluxio.host",
+            "hive.hadoop2.alluxio.port"
+    })
+    @BeforeClass
+    public void setup(String host, String port)
+    {
+        this.alluxioAddress = host + ":" + port;
+        System.out.println(this.alluxioAddress);
+        System.setProperty(PropertyKey.Name.SECURITY_LOGIN_USERNAME, "presto");
```

Comment:
Can you avoid setting system properties? Is there a more API-friendly way of passing this?

Reply:
How does the Alluxio code that uses these properties get run? Why can’t we configure it directly?

Reply:
The Alluxio client code that uses these properties runs as part of this test. 
In this test, the Presto client will trigger a call to the Alluxio connector, which will call Alluxio client, which will need these configuration values to contact alluxio master. 

Alluxio client can either receive configuration through environment variables or a configuration file , as explained here . (https://docs.alluxio.io/ee/user/stable/en/operation/Configuration.html?#configuration-sources)  Alternatively, we can output a configuration file, but I don't see that as a cleaner alternative.  Let me know if you have other suggestions. 

Thanks.